src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+public class Book {
+    private long id;
+    private String title;
+    private int price;
+
+    private Book(long id, String title, int price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
It's not consistent to use both ways of initialization of an object. Since you have setters for all fields, you can remove the private constructor and create objects using setters or create a public constructor that initializes all fields.